src/index.js

```diff
@@ -0,0 +1,108 @@
+import './style.css';
+import Icon from './option.png';
+import validateForm from './funct.js';
+import removeTask from './remove.js';
+import removeAllTask from './removeAll.js';
```

Comment:
Since all of these files are in the same directory, you could do 
` import {
  validateForm, removeTask, removeAllTask
};`

Reply:
According to requirements, the more files you have more interesting it'll. 
Even they are from the same directory but in different files.